examples/tv-app/android/java/AppImpl.cpp

```diff
@@ -403,6 +403,30 @@ Access::Privilege ContentAppFactoryImpl::GetVendorPrivilege(uint16_t vendorId)
     return Access::Privilege::kOperate;
 }
 
+constexpr ClusterId kClusterIdDescriptor = 0x001d;
+constexpr ClusterId kClusterIdOnOff      = 0x0006;
+constexpr ClusterId kClusterIdWakeOnLAN  = 0x0503;
+// constexpr ClusterId kClusterIdChannel             = 0x0504;
+// constexpr ClusterId kClusterIdTargetNavigator     = 0x0505;
+constexpr ClusterId kClusterIdMediaPlayback = 0x0506;
```

Comment:
Would it make sense to initialize using the IDs from the cluster definitions. So like
constexpr ClusterId kClusterIdMediaPlayback = chip::app::Clusters::MediaPlayback::Id;
Or directly use constants defined in cluster-id.h (#include <app-common/zap-generated/cluster-id.h>)
like ZCL_MEDIA_PLAYBACK_CLUSTER_ID?

What I want to avoid is the IDs being defined separately again. Although it is unlikely they will change.